tests/Feature/Concerns/HandlesAiFeatureResponsesTest.php

```
@@ -0,0 +1,162 @@
+    );
+
+    expect( $outcome )->toBeInstanceOf( AiFeatureOutcome::class )
+        ->and( $outcome->succeeded )->toBeTrue()
+        ->and( $outcome->feature )->toBe( 'ai.summarize' )
+        ->and( $outcome->output )->toBe( [ 'summary' => 'done' ] )
+        ->and( $outcome->status )->toBe( 200 )
+        ->and( $outcome->statusSlug )->toBe( 'success' )
+        ->and( $outcome->errorCode )->toBeNull()
+        ->and( $outcome->message )->toBeNull();
+
+    Log::shouldNotHaveReceived( 'error' );
```

PHPStan (2.2.8)

[error] Call to an undefined static method Illuminate\Support\Facades\Log::shouldNotHaveReceived().


(staticMethod.notFound)